go.mod

```diff
@@ -69,3 +69,5 @@ replace (
 	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.1.10
 	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.1.11-0.20190411181648-9d55346c2bde
 )
+
+go 1.13
```

Comment:
In this PR we should not include the change to make Go 1.13 as a requirement. I understand the new changes don't require Go 1.13.

Reply:
I didn't add this myself, I believe this is generated by the `go mod` command. I can remove it manually but I think if you upgrade `golang` to newer versions, this will come back. Do you still want me to manually remove it for now?

Reply:
Please remove it yes. We are not raising the minimum required Go version to 1.13 in this PR

Reply:
ok